feed/permissions.py

```diff
@@ -35,26 +36,39 @@ def has_permission(self, request, view):
             return True
         else:
             return False
-        return request.user and request.user.is_authenticated()
 
     def check_object_permission(self, user, obj):
         return (user and user.is_authenticated() and
-          (user.is_staff or obj == user))
+                (user.is_staff or obj == user))
 
     def has_object_permission(self, request, view, obj):
-
         user_level1 = getLevel1(request.user)
-        user_org = TolaUser.objects.get(user=request.user).organization
         if view.action == 'list':
-            return request.user.is_authenticated() and (obj in user_level1 and obj.organization in user_org
-                                                or 'OrgAdmin' in request.user.groups.values_list('name', flat=True))
+            user_org = TolaUser.objects.get(user=request.user).organization
+            return (request.user.is_authenticated() and
+                    (obj in user_level1 and obj.organization in user_org
```

Comment:
A TolaUser can be part of only one org, so it's better to use '==' or 'is'. More readable.

Reply:
That’s not technically true.  A tolauser isa member of the TolaData org and one other org.  That should be a many to many relationship.

Reply:
Yes, you're right. So, we have to change the field from `ForeignKey` to `ManyToMany` in the model.